java/src/com/liferay/nativity/Constants.java

```diff
@@ -45,7 +45,7 @@
 	public static final String MENU_TEXT = "menuText";
 
 	public static final String NATIVITY_REGISTRY_KEY =
-		"SOFTWARE\\Liferay Inc\\Liferay Nativity";
+		"SOFTWARE\\Box Inc\\Box Desktop";
```

Comment:
Sorry, I didn't mean to open this PR before the other PR was merged in. I'll make sure I will maintain ordering/chaining properly next time